6.8.2: Add release notes (#4054)
sensu/sensu-docs: content/sensu-go/6.8/release-notes.md

---

## 6.8.2 release notes

**October 6, 2022** &mdash; The latest release of Sensu Go, version 6.8.2, is now available for download.

> Sensu Go 6.8.2

Reviewer: Need to add the synopsis here; you may have already requested it

Author: Oh shoot

Read the [upgrade guide][1] to upgrade Sensu to version 6.8.2.

> **IMPROVEMENTS:**

Reviewer: There are a couple more improvements:

- Agent log messages now include the check name when a check execution request fails.
- On Windows platforms, the Sensu Agent service now automatically restarts after failures.

Author: Added.

- The authentication module now logs successful (INFO) and unsuccessful (ERROR) login attempts.

> - Agent websocket connection logging includes backend entity name.

Reviewer: ```suggestion
- Agent websocket connection logging now includes backend entity name.
```

> - A sensu.io/output_truncated_bytes label is now applied to events when the check output has been truncated due to a check's max_output_size configuration.

Reviewer: ```suggestion
- When check output is truncated due to the [max_output_size][302] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.
```

Reviewer: If this change is accepted, you'll also need to add this link reference:

[302]: /sensu-go/6.8/observability-pipeline/observe-schedule/checks/#check-output-truncation-attributes

> - When check output is truncated due to the [max_output_size][303] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.

Reviewer: ```suggestion
- When check output is truncated due to the [max_output_size][302] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.
```

Reviewer: need to add this link reference:

`[302]: /sensu-go/6.8/observability-pipeline/observe-schedule/checks/#check-output-truncation-attributes`

**FIXES:**

> - ([Commercial feature][295]) In the web UI, the silence action button is now present in the entities page.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, restored the silence function on the Entities page.
```

> - ([Commercial feature][295]) In the web UI, newly created assets are displayed immediately upon creation.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, resource pages now automatically refresh after creating resources.
```

> - ([Commercial feature][295]) In the web UI, the "Raw" tab for Pipelines now displays the raw configuration.

Reviewer: ```suggestion
- ([Commercial feature][295]) The web UI now displays pipeline definitions under the **RAW** tab on individual pipeline resource pages.
```

> - ([Commercial feature][295]) In the web UI, the link for entity documentation now points to the correct link.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, corrected the link to the entity reference in the Edit Entity modal.
```

- ([Commercial feature][295]) In the web UI, errors displayed  when deleting and re-adding an asset from asset page have been addressed.

> - ([Commercial feature][295]) In the web UI, checks no  longer error when created  with leading white spaces.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, fixed the validation for resource names and array fields to prevent crashes.
```

> - ([Commercial feature][295]) In the web UI, the loading indicator no longer spins endlessly when loading resources.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, the configuration resource pages now show an empty list instead of an endless loading indicator for users who do not have the required permissions.
```

> - ([Commercial feature][295]) In the web UI, users no longer encounter an error if they do not have an explicitly set username.

Reviewer: ```suggestion
- ([Commercial feature][295]) In the web UI, fixed a bug that could cause a crash when an authorized user does not have an explicitly set username.
```

> - Fixed an issue where multi-expression exclusive "Deny" filters were not evaluated as described in the documentation.

Reviewer: ```suggestion
- Fixed an issue that prevented multi-expression, exclusive event filters set to `deny` from being evaluated properly.
```

---

## 6.8.2 release notes

**October 6, 2022** &mdash; The latest release of Sensu Go, version 6.8.2, is now available for download.

Sensu Go 6.8.2

Read the [upgrade guide][1] to upgrade Sensu to version 6.8.2.

**IMPROVEMENTS:**

- The authentication module now logs successful (INFO) and unsuccessful (ERROR) login attempts.
- Agent websocket connection logging now includes backend entity name.
- When check output is truncated due to the [max_output_size][302] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.

**FIXES:**

- ([Commercial feature][295]) In the web UI, restored the silence function on the Entities page.
- ([Commercial feature][295]) In the web UI, resource pages now automatically refresh after creating resources.
- ([Commercial feature][295]) The web UI now displays pipeline definitions under the **RAW** tab on individual pipeline resource pages.
- ([Commercial feature][295]) In the web UI, corrected the link to the entity reference in the Edit Entity modal.

> - ([Commercial feature][295]) In the web UI, errors displayed  when deleting and re-adding an asset from asset page have been addressed.

Reviewer: I can't figure out what this item refers to (`In the web UI, errors displayed  when deleting and re-adding an asset from asset page have been addressed.`). I see it in the changelog, but I can't find the corresponding issue or PR that makes sense.

- ([Commercial feature][295]) In the web UI, fixed the validation for resource names and array fields to prevent crashes.
- ([Commercial feature][295]) In the web UI, the configuration resource pages now show an empty list instead of an endless loading indicator for users who do not have the required permissions.
- ([Commercial feature][295]) In the web UI, fixed a bug that could cause a crash when an authorized user does not have an explicitly set username.

> - Fixed an issue that prevented multi-expression, exclusive event filters set to `deny` from being evaluated properly.

Reviewer: I suggest adding the keepalive item to the list of fixes (something like `- Modified keepalive startup so that etcd lease errors will not cause sensu-backend crashes.`)

Author: Change incoming

- Agent log messages now include the check name when a check execution request fails.

> - On Windows platforms, the Sensu Agent service now automatically restarts after failures.

Reviewer: I would categorize these as improvements rather than fixes

Reviewer: Sorry ^ that means the last two items currently listed:

```
- Agent log messages now include the check name when a check execution request fails.
- On Windows platforms, the Sensu Agent service now automatically restarts after failures.
```
